addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -34,8 +34,18 @@ def _get_move_applicability(self, move):
         self.ensure_one()
         if self.code != 'in_einvoice_1_03':
             return super()._get_move_applicability(move)
-
-        if move.is_sale_document() and move.country_code == 'IN' and move.l10n_in_gst_treatment in (
+        base_gst_and_nill_tags = (
```

Comment:
you could add add the records and use `.ids` once in the list comprehension

Reply:
yes - task 3360018